### PR TITLE
uploading assingments into /MI-10-MEAN/akshay/HTML/

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: akshay/HTML/assing5.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            year they struck a <span style = "background-color: rgb(226, 224, 75);"> partnership.</span> <br>
+            Working from their dorm rooms, they built a search engine that used links to determine the importance of<br>
+            individual pages on the <span style="color:blue;text-size-adjust: 20px;"> World Wide Web .</span> <br><br>
+            They called this search engine <span style = "background-color: rgb(226, 224, 75);"> Backrub.</span><br>
+        </p>
+        <hr>
+        <h2>Initial public offering</h2>
+        <p>
+            Google's initial public offering (IPO) took place five years later, on 19th August 2004.<br>
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for 30 20<br>
```
Comment (komal453) [inline akshay/HTML/assing5.html]: 20 should be underlined and use strike tag for 30.

**Review by komal453 — CHANGES_REQUESTED** [review]

File: akshay/HTML/assing5.html (hunk 1 above)
Comment (vishwaraj1706) [reply akshay/HTML/assing5.html]: In line number 27 Backrub should be italic

**Review by vishwaraj1706 — COMMENTED** [review]
In line number 27, Backrub should be italic

**Review by komal453 — APPROVED** [review]

File: akshay/HTML/assing5.html (hunk 1 above)
Comment (ghost) [reply akshay/HTML/assing5.html]: > In line number 27 Backrub should be italic

done

**Review by vishwaraj1706 — APPROVED** [review]

File: akshay/HTML/assing5.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            They called this search engine <span style = "background-color: rgb(226, 224, 75);"> <i>Backrub.</i></span><br>
+        </p>
+        <hr>
+        <h2>Initial public offering</h2>
+        <p>
+            Google's initial public offering (IPO) took place five years later, on 19<sup>th</sup> August 2004.<br>
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <s>30</s> <u>20</u><br>
+           years, until the year 2024.<br>
+        </p>
+        <center ><h2 style="color:crimson">Thank you!!</h2></center>
```
Comment (Pawan1005) [inline akshay/HTML/assing5.html:36]: need to use marquee tag here

**Pull request merged**